src/main/ngapp/src/app/chat-data.service.ts

```diff
@@ -16,16 +16,17 @@ export class ChatDataService {
   constructor(private authService: AuthService, private http: HttpClient) {}
 
   // Get the object associated with the currrent user.
-  getUser(): Observable<User[]> {
+  getUser(): Observable<User> {
     const localUser = JSON.parse(localStorage.getItem('user'));
-    return this.http.get<User[]>('/user?userId=' + localUser.uid);
+    return this.http.get<User>('/user?userId=' + localUser.uid);
   }
 
   // Create a new user object for the user.
   addUser(user: User): Observable<any> {
     return this.http.post<User>('/user', user);
   }
 
+  //
```

Comment:
comment marks added but not comments made on line 21 :)

Reply:
Comments added